README.md

```diff
@@ -1,21 +1,40 @@
-<p align="center">
+<!-- <p align="center">
```

Comment:
Noting that I'm commenting this image since it added quite a bit of extra scroll for the user to get to the README / docs / content. We can always re-add it at any time if we feel it should be there.

Reply:
I like this (removal of logo) — especially since:
1. we don't really use this logo anywhere else (i.e., it's not essential)
2. getting straight to the arch diagram is more important